src/ensembl/src/content/app/browser/zmenu/zmenuGenomeBrowserAdaptor.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * This is a temporary file to adapt data sent from genome browser with bpane-zmenu event
+ * to the needs of browser chrome (primarily as regards to the id format)
+ */
+
+import { parseFeatureId } from 'src/content/app/browser/browserHelper';
```

Comment:
I guess this can either be removed or commented out.

Reply:
Yep, the whole `zmenuGenomeBrowserAdaptor.ts` can actually be removed. Thanks for reminding!

Reply:
✔️